src/java/org/apache/cassandra/db/memtable/SkipListMemtable.java

```diff
@@ -170,6 +171,30 @@ public long partitionCount()
         return partitions.size();
     }
 
+    @Override
```

Comment:
Do we ever use SLMemtable?  I'm totally fine with just stubbing in "throw unsupported" for paths that only Apache cares about

(FTR I'm also totally fine with just deleting old, optional implementations but that's out of scope for this PR)

Reply:
I don't know. It is there so I implemented it. Honestly, I'm not very happy with the fact we have to iterate through rows to count them.... Consider this temporary.